prow/README.md

```diff
@@ -40,6 +40,7 @@ Note: versions specified in these announcements may not include bug fixes made
 in more recent versions so it is recommended that the most recent versions are
 used when updating deployments.
 
+ - *March 30, 2018* `updateconfig` format change. It now supports namespace. Please check the plugins help.
```

Comment:
Please provide details of the change here instead of redirecting. (Also plugin help does not include an info related to this change...)

Reply:
done